config.proto

```diff
@@ -120,6 +120,9 @@ enum TraceReporterType {
     // OpenTelemetry protobuf reporting format.
     // see https://github.com/open-telemetry/opentelemetry-proto
     OTLP = 2;
+
+    // Console reporting
+    CONSOLE = 3;
```

Comment:
Could we change the name to `LOGGING`. Logging is used by OTEL:
* https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter
* https://github.com/open-telemetry/opentelemetry-java/blob/50408d499f85d5761d0a5ed9bf9d77d5ff01fff5/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java#L48

Reply:
Cancelling approval for more discussion on this.

Reply:
I think Logging is confusing, it feels like we are exporting logs but we do export traces. CONSOLE is explicit as you will look into CONSOLE, also we are talking about this being something meant for debug. Do we just want to change it because OTEL says so?

Reply:
As we're built on top of otel, as a rule we should follow their conventions unless there is a compelling reason not too.  I'm not sure this case warrants deviating from the convention.

Also, personally I think LOGGING is clearer than CONSOLE.

Reply:
+1. I would rather keep with sync with OTEL, bc HT is basically OTEL distribution. Using OTEL naming convention allows users to quickly understand HT config.